server/channels/store/localcachelayer/channel_layer.go

```diff
@@ -452,7 +457,12 @@ func (s LocalCacheChannelStore) SaveMultipleMembers(members []*model.ChannelMemb
 		return nil, err
 	}
 	for _, member := range members {
-		s.InvalidateMemberCount(member.ChannelId)
+		// For redis, directly increment member count.
+		if externalCache, ok := s.rootStore.channelMemberCountsCache.(cache.ExternalCache); ok {
+			s.rootStore.doIncrementCache(externalCache, member.ChannelId, 1)
```

Comment:
While changing the method may be out of scope, we could create a temporary map to avoid some roundtrips on the increment call. Of course I am assuming members would generally belong to the same channel.

Reply:
> Of course I am assuming members would generally belong to the same channel.

Yeah, that's the main thing. Atleast for the load-test, we add only one member at a time. Let me add a quick comment so that the context is there for future improvement.